src/NHibernate.Test/Hql/Ast/BulkManipulation.cs

```diff
@@ -1,28 +1,21 @@
 using System;
-using System.Collections;
+using System.Linq;
 using System.Threading;
-using NHibernate.Dialect;
 using NHibernate.Hql.Ast.ANTLR;
-using NHibernate.Id;
-using NHibernate.Persister.Entity;
+using NHibernate.Linq;
 using NUnit.Framework;
 
 namespace NHibernate.Test.Hql.Ast
 {
 	[TestFixture]
 	public class BulkManipulation : BaseFixture
```

Comment:
Heavily rewritten, as it was already done in #391: proper setup&teardown allowing to handle the "no temp table" case in which MsSqlCe falls.